src/app/pages/StakingPage/Features/ValidatorList/index.tsx

```diff
@@ -74,6 +75,11 @@ export const ValidatorList = memo((props: Props) => {
             <ShortAddress address={datum.address} />
           </Text>
         ),
+      sortable: true,
+      sortFunction: (row1, row2) =>
+        row1.name
+          ? row1.name.localeCompare(row2.name ?? row2.address)
+          : row1.address.localeCompare(row2.name ?? row2.address),
```

Comment:
```suggestion
      sortFunction: (row1, row2) => (row1.name ?? row1.address).localeCompare(row2.name ?? row2.address),
```
is more symmetric

Reply:
agreed with this suggestion above

could you also confirm that `??` is appropriate? surrounding code, e.g. `datum.name ? ( ... ) : ( ... )`, suggests that `&&` would be consistent

Reply:
Thanks, I applied the suggested change and also used `??` for `datum.name ? ( ... ) : ( ... )` above.